src/woo_publications/publications/migrations/0008_publication_opsteller_publication_publisher_and_more.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 4.2.16 on 2024-11-07 10:39
+
+from django.db import migrations, models
+from django.db.migrations.state import StateApps
+import django.db.models.deletion
+
+
+# The publisher field is a required fk field linked to the organisations model.
+# This means that for the migration to succeed we need to add a temporary default value.
+# Because of this we have made the temporary value 1, which translates to the organisation object with an id of 1.
+# If the database is empty or filled with organisations then this will go fine.
+# But in the scenario where we have existing publications but no organisations the migration will crash.
+# Because of this we created this function to fix this issue for us.
+# In the situation where we do have publications but no organisations this function creates a organisation to link the publications to.
+def create_organisation_if_none_exist(apps: StateApps, _):
+    Organisation = apps.get_model("metadata", "organisation")
+    Publication = apps.get_model("publications", "publication")
+
+    if Publication.objects.exists():
+        if not Organisation.objects.exists():
+            Organisation.objects.create(
+                naam="DELETE ME",
+                oorsprong="zelf_toegevoegd",
+                is_actief=True,
+            )
```

Comment:
there's no guarantee that existing organisation will have ID 1 - it may have been created and deleted at some point and the auto sequence just keeps incrementing

you could do `Organisation.objects.get_or_create(id=1, defaults={...})`